monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md

```diff
@@ -0,0 +1,78 @@
+# La Herramienta para el Renacimiento  
+
+**“Participa en un movimiento para un cambio positivo con una herramienta que le devuelve el poder social y económico a las personas: Monero.”**  
+8 de Junio del 2020  
+
+Es el año 2020, y estamos viendo cómo se desarrolla una crisis con diversos aspectos. Tenemos una pandemia global, el resurgimiento de tensiones raciales y económicas, la división geopolítica extrema y el abuso de poder por parte de gobiernos y bancos centrales de todo el mundo. Tenemos, nos dice la historia, los ingredientes para un cambio transformador. Si apoyas o estás entre los que están siendo oprimidos por razones raciales, religiosas, políticas o económicas, te invitamos a participar en un movimiento para un cambio positivo con una herramienta que le devuelve el poder social y económico a la gente: Monero.  
+
+El descontento y el malestar civil son a menudo los precursores de un cambio profundo. Los disturbios en la antigua Roma presagiaron etapas del fin de la república. En Estados Unidos, la independencia vino después de las protestas, daños en la propiedad y las respuestas violentas de las autoridades en contra de los manifestantes (tal como en la masacre de Boston). La protesta y la desobediencia civil acompañaron fuertes cambios en los derechos civiles a comienzos de 1960.  
+
+Los sistemas financieros a menudo están inextricablemente vinculados a los disturbios civiles y al cambio social. Las bases de la Revolución Americana incluyeron el deseo de la colonia por controlar los impuestos y el destino económico.  
+
+A medida que los bancos centrales comenzaron a imprimir dinero después del desmantelamiento total del patrón oro en 1971, nace un nuevo sistema monetario por decreto o sistema fiat que se convertiría en el eje monetario global, el dólar estadounidense (USD) determinaría el valor de la mayoría de las otras monedas. Las personas ahora están viendo las consecuencias indeseadas de este sistema financiero, croncretamente hablando de la fragilidad socioeconómica, la inequidad y la disparidad. Las persona quieren algo mejor que equilibre el poder. Esta vez, no necesitamos una guerra para lograrlo. Podemos optar simplemente salirnos pacíficamente escogiendo usar un sistema financiero tecnológico que está siendo desarrollado por y para las personas.  
```

Comment:
...este sistema financiero, "croncretamente" hablando..

Concretamente

Reply:
Do you want me to put double quotes to the word "_concretamente_"?

Reply:
Oh sorry, I see it's a typo. Will fix it right away.